.github/workflows/merge-commits.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           token: ${{ secrets.BOT_MAIN_NEXT_WORKFLOW_PAT }}
           script: |
             const { Octokit } = require("@octokit/core");
-            const token = `${{ secrets.GITHUB_TOKEN }}`;
+            const token = `${{ secrets.FLUIDBOTPAT }}`;
```

Comment:
We are already using `BOT_MAIN_NEXT_WORKFLOW_PAT` which is PAT. maybe replacing the GITHUB_TOKEN would work?